packages/@extension-kid/toast-notifications/ToastNotification.ts

```diff
@@ -0,0 +1,79 @@
+import { Notification } from "@extension-kid/notification-service";
+
+enum ToastAppearance {
+  Default = "default",
+  Success = "success",
+  Warning = "warning",
+  Danger = "danger"
+}
+
+enum ToastCallbackType {
+  Dismiss,
+  Primary,
+  Secondary
+}
+
+type ToastCallback = (
+  callbackType: ToastCallbackType,
+  notification: ToastNotification
+) => void;
+
+class ToastNotification extends Notification {
+  static NotificationType = Symbol("Toast");
+  readonly title: string;
+  readonly description: string;
+  readonly appearance: ToastAppearance;
+  readonly autodismiss: boolean;
+  readonly callback: ToastCallback | null;
+  readonly primaryActionText: string;
+  readonly secondaryActionText: string;
+
+  constructor(
+    title: string,
+    description: string = "",
+    callback: ToastCallback | null = null,
+    appearance: ToastAppearance = ToastAppearance.Default,
+    primaryActionText: string = "",
+    secondaryActionText: string = "",
+    autodismiss: boolean = false
```

Comment:
Our current blueprint-Toast-component does not use it, but we want it to be here in case we need it one day, right?

also there seems to be no test relying on it yet.

Reply:
the `Toast` in `ui-kit` supports this and the prop is set when creating the component.